README.markdown

```diff
@@ -264,6 +268,7 @@ The example above demonstrates the following style guidelines:
  + Define multiple variables and structures on a single line if they share a common purpose / context.
  + Indent getter and setter definitions and property observers.
  + Don't add modifiers such as `internal` when they're already the default. Similarly, don't repeat the access modifier when overriding a method.
+ + Every property is defined on it's own line; for example `x` and `y`don't share the line.
```

Comment:
Missing space after `y`
I'm also not sure, should there be a comma after "for example"?

Reply:
Maybe just:
Every property is defined on it's own line. For example `x` and `y` don't share the line.

@icecoffin ?